cmd/queue/main.go

```diff
@@ -109,10 +108,12 @@ func connectStatSink() {
 		servingAutoscaler, system.Namespace, servingAutoscalerPort)
 	logger.Infof("Connecting to autoscaler at %s.", autoscalerEndpoint)
 
-	wait.ExponentialBackoff(wait.Backoff{
+	// Will retry connecting to the autoscaler for roughly 1.5 minutes.
+	err := wait.ExponentialBackoff(wait.Backoff{
 		Duration: 100 * time.Millisecond,
 		Factor:   1.3,
-		Steps:    math.MaxInt32,
+		Steps:    20,
+		Jitter:   0.5,
```

Comment:
Confirming the intent here: According to https://godoc.org/k8s.io/apimachinery/pkg/util/wait#ExponentialBackoff, this will cause the first sleep to be between 100ms and 150ms. Following sleeps will be between 100% and 150% of the indicated duration.

Reply:
Nope, seems fine to me.